### DISCUSSION TITLE
Could not load type 'Microsoft.Graph.HttpProvider' from assembly 'Microsoft.Graph.Core, Version=3.0.10.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35'.

### DISCUSSION THREAD

**Discussion by orbcom-tiagosantos** [pnp/powershell#3387] [Q&A]
### Discussed in https://github.com/pnp/powershell/discussions/3380

<div type='discussions-op-text'>

<sup>Originally posted by **TiagoSantosOrbcom** September  1, 2023</sup>
I have a solution develop in .NET 7 C# that creates sharepoint sites via Microsoft.Powershell.SDK, after the site has been created, i extract site template from a site model, when i tried to do that this error occurred.

Nugget Package:
> Microsoft.Graph (5.25.0)
> Microsoft.Kiota.Abstractions (1.3.1)
> Microsoft.PowerShell.SDK (7.3.6)
> PnP.Core.Auth (1.10.0)
> PnP.Framework (1.13.0)
> System.Management.Automation (7.3.6)

PowerShell Module:
> PnP.Framework (2.1.1)
> Microsoft.Graph (2.4.0)

PowerShell Command
> Get-PnPSiteTemplate -Out C:\\temp\\teste.pnp -Handlers All -PersistBrandingFiles -PersistPublishingFiles -ExcludeHandlers "SiteSecurity" -Force

Can some on help me please</div>

**Comment by MathieuMarchant** [comment]
I'm bumping into the same issue. Were you able to resolve this?
